src/components/Lend/LoanSearch/TeamPicksSwitch.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+	<div class="tw-bg-white tw-rounded tw-p-3 lg:tw-shadow-lg">
+		<div class="tw-flex tw-flex-col">
+			<kv-switch
+				v-model="showTeamPicks"
+			>
+				<div class="tw-flex tw-items-center tw-gap-0.5">
+					<img v-if="HandOrangeIcon" :src="HandOrangeIcon" class="tw-w-4">
```

Comment:
Is `v-if="HandOrangeIcon"` necessary here?

Reply:
Not really needed, a good practice if the icon is removed by a bad practice...